src/Site_WP_Docker.php

```diff
@@ -178,7 +178,7 @@ public function generate_docker_compose_yml( array $filters = [] ) {
 
 		$base[] = $php;
 		$base[] = $nginx;
-		//$base[] = $mailhog;
+		$base[] = $mailhog;
```

Comment:
Is this for specifying the container that will be run at the start? Mailhog shouldn't be enabled by default.